src/yetibot/api/google.clj

```diff
@@ -0,0 +1,83 @@
+(ns yetibot.api.google
+  (:require
+   [taoensso.timbre :refer [warn info]]
+   [clojure.string :as s]
+   [clojure.data.json :as json]
+   [clj-http.client :as client]
+   [yetibot.core.config :refer [conf-valid? config-for-ns]]))
+
+(def config (config-for-ns))
+
+(defn configured? []
+  (let [config-keys   [:api-key :custom-search-engine-id]
+        config-values (select-keys config config-keys)]
+    (and (conf-valid? config)
+       (= (count config-values) 2))))
+
+(defonce api-url "https://www.googleapis.com/customsearch/v1?parameters")
+
+;; this has yet to be used somewhere
+(defonce keywords (atom {}))
```

Comment:
Makes sense. I think for things like safe search, page, sort, image size, etc, jira-style opts would work well. A bunch of overloads for all those options would be pretty confusing and hard to maintain.

Can either:

1. start with simple impl where user's only get to configure `lang` (and anything else) in config.edn then maybe add the rest later
2. do 1 and also provide jira-style opts for the rest

Thoughts?